pkg/storage/utils/localfs/localfs_windows.go

```diff
@@ -18,7 +18,7 @@
 
 // +build windows
```

Comment:
the build tad is redundant, as your filename contains `_windows` on it

Reply:
@refs Makes sense. I'll remove the tags then. Thanks!